mlflow/pyfunc/model.py

```diff
@@ -243,6 +243,28 @@ def predict(self, context, messages: List[ChatMessage], params: ChatParams) -> C
             the model's response(s), as well as other metadata.
         """
 
+    def predict_stream(
+        self, context, messages: List[ChatMessage], params: ChatParams
+    ) -> Iterator[ChatResponse]:
+        """
+        Evaluates a chat input and produces a chat output.
+        Overrides this function to implement a real stream prediction.
+
+        Args:
+            messages (List[:py:class:`ChatMessage <mlflow.types.llm.ChatMessage>`]):
+                A list of :py:class:`ChatMessage <mlflow.types.llm.ChatMessage>`
+                objects representing chat history.
+            params (:py:class:`ChatParams <mlflow.types.llm.ChatParams>`):
+                A :py:class:`ChatParams <mlflow.types.llm.ChatParams>` object
+                containing various parameters used to modify model behavior during
+                inference.
+
+        Returns:
+            An iterator over :py:class:`ChatResponse <mlflow.types.llm.ChatResponse>` object
+            containing the model's response(s), as well as other metadata.
+        """
+        return iter([self.predict(context, messages, params)])
```

Comment:
```suggestion
        yield self.predict(context, messages, params)
```

can we use `yield` here?